redbot/cogs/trivia/trivia.py

```diff
@@ -512,6 +568,66 @@ def get_trivia_list(self, category: str) -> dict:
             else:
                 return dict_
 
+    async def _save_trivia_list(self, ctx: commands.Context, file: discord.Attachment):
+        """Checks and saves a trivia list to data folder.
+
+        Parameters
+        ----------
+        file : discord.Attachment
+            A discord message attachment.
+
+        Returns
+        -------
+        None
+        """
+        filename = file.filename.rsplit(".", 1)[0]
+
+        buffer = io.BytesIO(await file.read())
+        yaml.safe_load(buffer)
+        buffer.seek(0)
+
+        file = cog_data_path(self) / f"{filename}.yaml"
+
+        basefileexists = None
+
+        if filename in ("upload", "delete"):
+            basefilexists = True
+        elif any(filename == item.stem for item in get_core_lists()):
+            basefileexists = True
+
+        if file.exists() or basefileexists:
+            if basefileexists:
+                await ctx.send(
+                    _(
+                        "{filename} is a reserved trivia name and cannot be replaced.\n"
+                        "Choose another name".format(filename=filename)
+                    )
+                )
+                return
+            else:
+                await ctx.send(
+                    _(
+                        "{filename} already exists. Do you wish to overwrite?".format(
+                            filename=filename
+                        )
+                    )
```

Comment:
String has to be formatted after translation.
```suggestion
                    _(
                        "{filename} already exists. Do you wish to overwrite?"
                    ).format(filename=filename)
```